backtrace-library/src/main/java/backtraceio/library/services/BacktraceHandlerThread.java

```diff
@@ -83,7 +85,18 @@ public void handleMessage(Message msg) {
             BacktraceResult result;
             if (mInput.requestHandler != null) {
                 BacktraceLogger.d(LOG_TAG, "Sending using custom request handler");
-                result = mInput.requestHandler.onRequest(mInput.data);
+                boolean containsMinidump = false;
+                for (String attachment: mInput.data.getAttachments()) {
```

Comment:
extract to separate method in file `BacktraceNativeData` e.g. with name containsMinidumpFile(..)